### ISSUE TITLE
Comman line pushed out of screen

### ISSUE THREAD

**Issue by artkpv** [equalsraf/neovim-qt#574]
Hi, thanks for this tool!
How to make it show my command line at the bottom when I make it full screen? Please see the screenshot. Under ArchLinux, KDE Plasma 
```
[art@arch-pc ~]$ nvim --version
NVIM v0.3.8
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/bin/cc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wconversion -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -Wno-array-bounds -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim/src/build/config -I/build/neovim/src/neovim-0.3.8/src -I/usr/include -I/build/neovim/src/build/src/nvim/auto -I/build/neovim/src/build/include
Compiled by builduser

Features: +acl +iconv +jemalloc +tui 
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

[art@arch-pc ~]$ sudo pacman -Qs neovim-qt
local/neovim-qt 0.2.12-2
    GUI for Neovim

[art@arch-pc ~]$ plasmashell --version
plasmashell 5.16.4

```

![image](https://user-images.githubusercontent.com/3657998/63223583-84f45f00-c1c0-11e9-816f-9212faf22d3d.png)

**Comment by jgehrig** [comment]
Thanks for the bug report! I can reproduce this issue with `:GuiTabline 1`.

As a workaround, try setting `:GuiTabline 0` in your `~/.config/nvim/ginit.vim`:
```
" Use Vim Styling for Tabs, Qt Disabled
if exists(':GuiTabline')
	GuiTabline 0
endif
```

It looks like the window resize code isn't properly handling the height of GuiTabline. I will investigate the issue.

For reference, my system has the following configuration:
- Gentoo
- High DPI Display
- kde-plasma/plasma-meta-5.15.5
- app-editors/neovim-0.3.7
- app-editors/neovim-qt-0.2.12
- dev-qt/qtcore-5.12.3

**Comment by artkpv** [comment]
Confirm. It is fixed in 0.2.13
